src/python/test/test_cubical_complex.py

```diff
@@ -152,8 +152,8 @@ def test_dimension_file_constructor():
     assert cub.__is_persistence_defined() == False
     assert cub.persistence() == [(1, (0.0, 100.0)), (0, (0.0, float("inf")))]
     assert cub.__is_persistence_defined() == True
-    assert cub.betti_numbers() == [1, 0, 0]
-    assert cub.persistent_betti_numbers(0, 1000) == [1, 0, 0]
+    assert cub.betti_numbers() == [1, 0]
+    assert cub.persistent_betti_numbers(0, 1000) == [1, 0]
```

Comment:
Ah, yes, I forgot it would have this kind of consequence. If we wanted to, we could just append a `0` to keep the same output. I don't know what is better...